osm-utils/src/main/java/org/orbisgis/osm_utils/overpass/OverpassStatus.groovy

```diff
@@ -0,0 +1,182 @@
+ * OrbisGIS is developed by the GIS group of the DECIDE team of the
+ * Lab-STICC CNRS laboratory, see <http://www.lab-sticc.fr/>.
+ *
+ * The GIS group of the DECIDE team is located at :
+ *
+ * Laboratoire Lab-STICC – CNRS UMR 6285
+ * Equipe DECIDE
+ * UNIVERSITÉ DE BRETAGNE-SUD
+ * Institut Universitaire de Technologie de Vannes
+ * 8, Rue Montaigne - BP 561 56017 Vannes Cedex
+ *
+ * OSM is distributed under LGPL 3 license.
+ *
+ * Copyright (C) 2019 CNRS (Lab-STICC UMR CNRS 6285)
+ *
+ *
+ * OSM is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * OSM is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with
+ * OSM. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For more information, please consult: <http://www.orbisgis.org/>
+ * or contact directly:
+ * info_at_ orbisgis.org
+ */
+package org.orbisgis.osm_utils.overpass
+
+import java.text.SimpleDateFormat
+import java.time.ZoneId
+
+/**
+ * Status of the overpass server.
+ *
+ * @author Sylvain PALOMINOS (UBS LABSTICC 2019)
+ */
+class OverpassStatus {
+
+    /** String used to parse the slot {@link java.lang.String} representation */
+    private static final CONNECT_AS = "Connected as: "
+    /** String used to parse the slot {@link java.lang.String} representation */
+    private static final CURRENT_TIME = "Current time: "
+    /** String used to parse the slot {@link java.lang.String} representation */
+    private static final RATE_LIMIT = "Rate limit: "
+    /** String used to parse the slot {@link java.lang.String} representation */
+    private static final SLOT_AVAILABLE = " slots available now."
+    /** String used to parse the slot {@link java.lang.String} representation */
+    private static final RUNNING_QUERIES = "Currently running queries (pid, space limit, time limit, start time):"
+
+    /** {@link java.text.SimpleDateFormat} used to parse dates. */
+    private format = "yyyy-MM-dd'T'HH:mm:ss'Z'" as SimpleDateFormat
+    private local = "yyyy-MM-dd'T'HH:mm:ss'Z'" as SimpleDateFormat
+
+    /** Connection id */
+    def connectionId
+    /** {@link java.util.Date} of the status */
+    def time
+    /** Maximum number of slots */
+    def slotLimit
+    /** Number of available slots */
+    def slotAvailable = 0
+    /** Lis of locked slots */
+    def slots = []
+    /** List of running queries */
+    def runningQueries = []
+
+    /**
+     * Main constructor
+     * @param test String representation of the server status.
+     */
+    OverpassStatus(String test){
```

Comment:
Is it possible to set a more understandable name ?

Reply:
Changed to `queryStatus`